scripts/get_counts.py

```diff
@@ -411,6 +413,16 @@ def MergeDevData(dest_count_dir, ngram_order):
     log_file = dest_count_dir + '/log/merge_dev_counts.log'
     RunCommand(command, log_file, args.verbose=='true')
 
+# this function get the integer in max_memory formatted as '10G' or '1024'
+def ParseInt(string):
+    if string != '':
+      return int(''.join([x for x in string if x.isdigit()]))
+
+# this function get the arbitary string (letters or %) at the end of max_memory
+# For example, 'G', 'b', '%' in '10G', '10b', and '10%'
+def ParseChar(string):
+    if string != '':
+      return str(''.join([x for x in string if not x.isdigit()]))
```

Comment:
I think it would be better, we use a single function to parse the string, and return the number and unit together. somthing like this:

def ParseMemoryString(string):
    ......
    return (number, unit)